### PR TITLE
Improves loot drop spawners, adds AI law spawners to cores

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
@Okand37 @pubby @ShizCalev 

The map changelogs are kind of really spammy if i write them into the changelog proper, so here they are

delta & omega:
* replaced paladin and corp boards with a "harmless" 3 law spawner (may spawn a duplicate asimov in the core)
* moved asimov onto the same tile with default and freeform core modules (one tile down)
* replaced drone, reporter and liveandletlive with a "neutral" 3 law spawner
* replaced antimov and tyrant with a "harmful" 2 law spawner
* moved protectstation one tile down to chill with purge and onehuman

box, meta & pubby:
* replaced corp with a "neutral" spawner
* replaced antimov with a "harmful" spawner
* added a "harmless" spawner to the core modules "locker". This can be a duplicate asimov.

[Changelogs]: 

:cl: Naksu
code: loot drop spawners now assign their pixel offsets to the items they spawn, also have a "fanout" setting to distribute items in a neat fashion like in omega/delta's core and some tech storages.
tweak: replaced several law boards in uploads with spawners, the net result being one more board in uploads that may be a duplicate asimov (but hopefully isn't)
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
>paladin
>harmless
>>still being in the game

**Comment by vuonojenmustaturska** [comment]
i asked about it when i added the spawner objects, it's supposed to be harmless and bwoinkable if the usual paladin AI things start happening

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -54298,12 +54290,8 @@
 	name = "Core Modules";
 	req_access_txt = "20"
 	},
-/obj/item/aiModule/core/full/antimov{
-	pixel_x = 3;
-	pixel_y = 3
-	},
-/obj/item/aiModule/supplied/oxygen,
-/obj/item/aiModule/supplied/protectStation{
+/obj/effect/spawner/lootdrop/aimodule_harmful/two,
```
Comment (ShizCalev) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: Just edit the vars instead of making new types for these.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by vuonojenmustaturska** [comment]
gave it a re-test and it (still) works after fixing that ,/; goof-up that no-one saw

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**